### DISCUSSION TITLE
Issues with git branches

### DISCUSSION THREAD

**Discussion by almirsarajcic** [3Stones-io/omedis#332] [General]
@amos-kibet, I see you were pushing some changes into the wrong branch (https://github.com/wintermeyer/omedis/pull/316 instead of https://github.com/wintermeyer/omedis/pull/329).
Let me know if you have questions. I'm not sure how this happened, but if you explain what you were trying to achieve, maybe I can tell you how to do it.

Also, have you tried GitButler? Maybe that can help you avoid this in the future.

**Comment by amos-kibet** [comment]
I made a mistake and pushed code belonging to a different branch. I fixed it by reverting the commits, and solved the issue.

**Reply by almirsarajcic** [reply]
Ok. I guess being more careful will prevent this in future.
